tests/main/interfaces-contacts-service/task.yaml

```diff
@@ -20,6 +20,12 @@ environment:
     XDG_DATA_HOME: $XDG/share
     XDG_CACHE_HOME: $XDG/cache
 
+debug: |
+    echo "Output process to see what might write to $XDG"
+    ps uafx
+    echo "Show what is in $XDG"
+    ls -alR "$XDG"
```

Comment:
Would it be useful to show `lsof` output here (possibly filtered by grep "$XDG")?

Reply:
Probably, I hope we can remove this debug stanza again soon once we figure out what is going on with the failure. But a good idea